optimize/pom.xml

```diff
@@ -55,6 +56,8 @@
     <version.node>v20.9.0</version.node>
     <version.yarn>v1.22.19</version.yarn>
 
+    <!-- lucene config-->
+    <lucene.version>8.11.1</lucene.version>
```

Comment:
is the version that is used here important? Note that in this repository it is likely that this gets updated automatically. If this got bumped to version 9, would that cause problems?

Reply:
is the version that is used here important? - version is not important.

Reply:
And an automatic update to 9.x won't break things?

Reply:
tested yes we will not have any problems (also I removed unused dependency)

Reply:
Created [this issue](https://github.com/camunda/camunda/issues/22303) and assigned it to you for now